### DISCUSSION TITLE
Cumulocity LoRa framework discussion board

### DISCUSSION THREAD

**Discussion by cpoder** [SoftwareAG/cumulocity-lora#115] [Announcements]
## 👋 Welcome!
You can use this discussion to:
* ask questions about the framework
* ask for assistance
* ask for specific requirements

If you need to contact me privately, you can do so by dropping me an email on cyril.poder(at)gmail.com

**Comment by phcrb** [comment]
[like]  Philippe Crabier reacted to your message:
________________________________
From: Cyril PODER ***@***.***>
Sent: Monday, July 15, 2024 11:59:32 AM
To: SoftwareAG/cumulocity-lora ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [SoftwareAG/cumulocity-lora] Cumulocity LoRa framework discussion board (Discussion #115)


👋 Welcome!

You can use this discussion to:

  *   ask questions about the framework
  *   ask for assistance
  *   ask for specific requirements

If you need to contact me privately, you can do so by dropping me an email on cyril.poder(at)gmail.com
